.travis/klee.sh

```diff
@@ -3,6 +3,22 @@
 set -e
 : ${SOLVERS?"Solvers must be specified"}
 
+coverageup()
+{
+  # Create report
+  # (NOTE: "--rc lcov_branch_coverage=1" needs to be added in all calls, otherwise br coverage gets dropped)
+  lcov --rc lcov_branch_coverage=1 --directory . --capture --output-file coverage.info
+  # Exclude uninsteresting coverage goals (e.g. LLVM)
```

Comment:
misspelled

Reply:
I would add a bit of explanation about each exclusion.  What is the `test-utils` directory?